_posts/one-list-to-rule-them-all.md

```diff
@@ -0,0 +1,74 @@
+2. Create a new email.
+3. Paste the link.
+4. Go back to Slack and copy the message text.
+5. Past it in the mail body.
+6. Send the message to me.
+
+Too much friction in creating a to-do list item.
+
+As a workaround, I used the Slack saved items feature to keep track of things: if I needed to follow up on something in Slack, I saved the Slack message, and the saved items were my things to do. That lacked an important thing. There is no way to add context to what was saved. And it's not only that, the only way to mark an item as done is to _unsave_ it. As soon as you do it, there is no way to undo the action and no easy way to find the message again. Over time, I lost many messages I needed to find a better solution.
+
+Last year I started using [Zapier](https://zapier.com) to automatically turn a saved Slack message into an email in my inbox. And that was quite an improvement. It works amazingly well, and it's cheap. I can use the free tier with the number of messages I save monthly.
+
+Using my email inbox as a to-do list is also handy on devices. I can tap share on a webpage on my phone, select Gmail, add some text, and click send. Boom, it's on my to-do list.
+
+One could argue why not use one of the many excellent to-do list applications. For example, Todoist does an excellent job, is flexible, and is highly customizable. My problem is that it would leave me with multiple to-do lists in different places. I already have in my inbox, for example, emails I receive from customers or support tickets or notifications from Dropbox Paper. I don't want to have to look in other places.
+
+There are two main reasons:
+
+- I have everything in one place, my Gmail inbox
+- I have everything in two places, my work Gmail inbox, my personal Gmail inbox
+
+Contradicting reasons, they seem! Somewhat :-)
+
+One of the disadvantages of the many to-do list Apps is that they don't have good multi-account support. One that comes with a good multi-account is Google Tasks, which also integrates very nicely with Gmail. However, I can't see any advantage over what I'm doing today. I tried Tasks twice for the records and returned to my trusted email twice.
+
+Gmail and many other email systems come with excellent support for multiple accounts. That allows me to have a clear separation between personal and work to-dos, which is also a nice-to-have if, like me, you don't have any work-related accounts on the phone or the tabled.
+
+Let me give you an example of a tool that doesn't come with a good enough separation of work and personal stuff and ends up being a source of frustration for me: GitHub Notifications (or the GitHub app for mobile devices). GitHub combines all issues and pull requests in a single place and provides little to no support to separate work-related things from personal ones, e.g., open source work. If one dedicates some time to open source activities during the weekend, they are also distracted by work-related issues and pull requests.
```

Comment:
You might need to explain that you use the same GitHub account for work and personal.

Reply:
```suggestion
Let me give you an example of a tool that doesn't come with a good enough separation of work and personal stuff and ends up being a source of frustration for me: GitHub Notifications (or the GitHub app for mobile devices). GitHub combines all issues and pull requests in a single place and provides little to no support to separate work-related things from personal ones, e.g., open source work. If one dedicates some time to open source activities during the weekend, they are also distracted by work-related issues and pull requests. Sure, I could be using two separate accounts, instead of one. The downside is that switching accounts requires logging off and logging in with a different one. Thanks, no, thanks.
```